app/src/main/java/richard/chard/lu/android/areamapper/SaveImageAsyncTask.java

```diff
@@ -69,12 +69,8 @@ protected Void doInBackground(Void... voids) {
 
         // Check for image folder, create if needed
 
-        File imageFolder = new File(
-                Environment.getExternalStoragePublicDirectory(
-                        Environment.DIRECTORY_PICTURES
-                ),
-                folderName
-        );
+        File imageFolder = new File(folderName);
```

Comment:
Where does this folder gets created on filesystem ? Is is still inside area mapper specific directory ?